ManualTimeLogger.Domain/DurationParser.cs

```diff
@@ -15,9 +16,18 @@ public ParseResult<float> Parse(string input)
         {
             var decimalSeparator = CultureInfo.CurrentCulture.NumberFormat.NumberDecimalSeparator;
             var durationString = _selector.Get(input).InputPart;
-            var parseResult = float.TryParse(durationString?.Replace(",", decimalSeparator).Replace(".", decimalSeparator), out var duration);
+            if (durationString?.Contains(":") ?? false)
+            {
+                var parseResult = TimeSpan.TryParse(durationString, out var timeSpan);
```

Comment:
```suggestion
            if (TimeSpan.TryParse(durationString, out var timeSpan))
```

Reply:
Hmmm, dit is lastig omdat dan hele getallen als valide `TimeSpan` worden gezien, maar hij parst `4` als vier dagen ipv vier uur. Het werkt dus alleen als er nadrukkelijk een `:` in zit. Tenzij er ergens een instelling te vinden is om de standaard parsing in uren te doen ipv dagen, maar die heb met een snelle Google nog niet kunnen vinden.

Reply:
Helder. Niet aan gedacht.